### ISSUE TITLE
msg: "This server is not the current active instance"

### ISSUE THREAD

**Issue by JeffersonStarseed** [AzuraCast/AzuraCast#6942]
### Installation Method

Docker Installation

### AzuraCast Release Channel

Stable Channel

### Current AzuraCast Version

0.19.4 Stable • Docker • PHP 8.2

### What happened?

After doing an Azuracast update in Glish on Linode (as described by Buster on the Linode video on YT), and then logging back into Azuracast, I was hit with this message, "This server is not the current active instance" (see attached screenshot).

My thought is that it may be that this may have occurred when I added my domain name in place of the IP address for the primary URL.  It may have created the second instance when I installed my domain name into Azuracast, as my domain URL takes me to one instance, and the IP address takes me to the other instance.

![Azuracast Issue 1](https://github.com/AzuraCast/AzuraCast/assets/159277623/d72e1c43-1ba6-4260-a0db-fd56a4db4072)

Also, my CPU/memory seems to have increased on my Linode server.  How do I uninstall the one Azuracast instance that I don't want?

**Comment by BusterNeece** [comment]
@JeffersonStarseed This can sometimes show up right after an update or a restart, as the "quorum sensing" in the database thinks the previous running instance is the "active" one. It usually self-resolves after a minute or two and the other instance goes inactive.

If you actually do have two instances of AzuraCast running, restarting your server should clear the issue up and stop the other running instance, though it's very rare to wind up in that situation.

**Comment by JeffersonStarseed** [comment]
Thank you for your reply, and you were right.  ...and definitely good to know this.  Later on in the night I noticed I didn't get the message anymore when logging into Azuracast.  I have done 2 reboots of the server since then.  That being said however, ....I tested ...and it seems I can still logon to Azuracast under my domain name and also under my server IP.  I've yet to determine if they are truly separate instances for sure yet.
